Source/WebKit/UIProcess/Cocoa/UIDelegate.mm

```diff
@@ -239,7 +239,8 @@
     if (!delegate)
         return completionHandler(menu);
 
-    auto contextMenuElementInfo = adoptNS([[_WKContextMenuElementInfo alloc] init]);
+    auto apiHitTestResult = API::HitTestResult::create(data);
+    auto contextMenuElementInfo = adoptNS([[_WKContextMenuElementInfo alloc] _initWithHitTestResult:wrapper(apiHitTestResult.get())]);
```

Comment:
I think this should look more like this:
auto contextMenuElementInfo = API::ContextMenuElementInfoMac::create(data);
then below we can use wrapper(contextMenuElementInfo)

Reply:
Done!